pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -287,6 +291,8 @@ func (p *DefaultProvider) generateNetworkInterfaces(options *amifamily.LaunchTem
 				// Instances launched with multiple pre-configured network interfaces cannot set AssociatePublicIPAddress to true. This is an EC2 limitation. However, this does not apply for instances
 				// with a single EFA network interface, and we should support those use cases. Launch failures with multiple enis should be considered user misconfiguration.
 				AssociatePublicIpAddress: options.AssociatePublicIPAddress,
+				PrimaryIpv6:              lo.Ternary(p.ClusterIPFamily == corev1.IPv6Protocol, lo.ToPtr(true), nil),
+				Ipv6PrefixCount:          lo.Ternary(p.ClusterIPFamily == corev1.IPv6Protocol, lo.ToPtr(int64(1)), nil),
```

Comment:
What's the rationale behind picking one here? is it to account for the primary? Mostly looking to be educated here.

Reply:
The idea is that the CNI allocates a prefix anyways once a pod tries to schedule to the node. So this is effectively saving time while doing exactly what CNI does